### ISSUE TITLE
Tried multiples all the steps but its not working.

### ISSUE THREAD

**Issue by junaideqbal** [itsMoji/Instagram_SSL_Pinning#79]
After applying the patch Instagram is still working and it's not giving any network error issue.
I'm following all the instructions as described.

**Comment by LearnJava** [comment]
> After applying the patch Instagram is still working and it's not giving any network error issue.
> I'm following all the instructions as described.

Non-root method do not work in my test. Root method working very good with Genymotion but do not work in Android Studio emulator devices.(IMHO . In my tests)
Need video step-by-step for this positions.

**Comment by junaideqbal** [comment]
@LearnJava I did tried the root method on Genymotion, But it's not working. after I applied the patch Instagram app still works and no error message is popping. Can you help me in making this work?

**Comment by LearnJava** [comment]
> @LearnJava I did tried the root method on Genymotion, But it's not working. after I applied the patch Instagram app still works and no error message is popping. Can you help me in making this work?

Try to use ARM part...

Usage
Download and install patched APK (ARM - x86)
ARM on a physical device or ARM on Genymotion Android 8-Oreo with ARM Translation strongly recommended!

1.Download ARM apk;
2.Download ARM_Translation_Oreo.zip from this link https://mega.nz/#F!JhcFwKpC!yfhfeUzvIZoSdBgfdZ9Ygg ;
3.Drug-n-drop ARM_Translation_Oreo.zip to your Gemnymotion android system (Oreo 8.0);
4.Reboot android(hold down the power button for some seconds);
4.Drug-n-drop Instagram APK to your android system(do not run this);
5.Watch video "How to install Burp cert to android" https://youtu.be/gmYzlpy2Ii4?t=435 (time 7:15)
6.Run Burp;
7.Setup proxy properties;
8.Save cert to file;
9.Drug-n-drop cert file to your android system;
10.Install cert (see step 5);
11.Setup proxy properties (ip and port) to android wi-fi channel;
12.Off interceptor in Burp;
13.Start Instagram app;
14.See requests in Burp;

That's all.

P.S. Sorry for my English )

**Comment by junaideqbal** [comment]
@LearnJava I tried your provided steps and **THANKS** these are working fine.
The only problem which I'm now facing is when I apply proxy of burp to ARM Genymotion Android. The request from app is sent and getting logged in burp but no response is coming back. I tried to load a web page in device, Same request is getting sent but no response. Now device don't have internet incoming. Because of this app is crashing. Any solution for this?

![image](https://user-images.githubusercontent.com/16623553/93339358-e9ab3e80-f844-11ea-90a2-a1b77cb8df31.png)
